pyqttoolkit/views/labelled.py

```diff
@@ -0,0 +1,30 @@
+# pyqttoolkit
+# Copyright (C) 2018-2019, Simmovation Ltd.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+from PyQt5.Qt import QWidget, QHBoxLayout, QLabel
+
+
+def make_labelled(parent, label: str, widget: QWidget):
```

Comment:
If you add lots of rows of these widgets, they won't line up on the horizontal axis. Assuming this is intentional, that's fine.

Reply:
if one is adding loads of consecutive rows, they should use a `QFormLayout`. This is really about fitting one of these amongst other widgets that don't conform to the form layout

Reply:
if fact I have added two of these consecutively and they did seem to line up. Probably due to an implicit size policy?

Reply:
Perhaps - I guess my point is they won't necessarily line up. If this isn't the goal of this then that's fine though